### PR TITLE
Doc: Fix escalation issue 903

### PR DESCRIPTION
**Pull request by achatterjee-grafana**
Changes include:
- Reorganized content for easier reading.
- Removed duplicate content.
- Introduced a table for the playlist mode content.

Two additional fixes not related to issue 903:
- Fixed a broken bookmark in Azure authentication topic.
- Updated Dashboard HTTP API endpoint. (https://raintank-corp.slack.com/archives/C5PG2JK8W/p1632753747127000?thread_ts=1632396502.122000&cid=C5PG2JK8W)

### CODE REVIEW

**Review by zuchka — APPROVED** [review]
nice!

**Review by GrafanaWriter — APPROVED** [review]
Thanks, @achatterjee-grafana

File: docs/sources/dashboards/playlist.md [hunk 1]
```diff
@@ -7,132 +7,83 @@ weight = 4
...
-1. Click **Create**.
+1. Hover your cursor over Grafana’s side menu.
+1. Click **Playlists**. You will see a list of existing playlists.
 
-## Edit a playlist
+### Start a playlist
 
-You can edit playlists while creating them or after saving them.
+1. [Access](#access-playlist) the playlist page to see a list of existing playlists.
+1. Find the playlist you want to start, then click **Start playlist**. The start playlist dialog opens.
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/dashboards/playlist.md:29]: Letting this stay as is. Hope that is OK :)

File: docs/sources/dashboards/playlist.md [hunk 2]
```diff
@@ -7,132 +7,83 @@ weight = 4
...
-| Back (left arrow)              | Returns to the previous dashboard.                                                                                                              |
-| Stop (square)                  | Ends the playlist, and exits to the current dashboard.                                                                                          |
-| Cycle view mode (monitor icon) | Rotates the display of the dashboards in different view modes.                                                                                  |
-| Time range                     | Displays data within a time range. It can be set to display the last 5 minutes up to 5 years ago, or a custom time range, using the down arrow. |
-| Refresh (circle arrow)         | Reloads the dashboard, to display the current data. It can be set to reload automatically every 5 seconds to 1 day, using the drop down arrow.  |
-
-> Shortcut: Press the Esc key to stop the playlist from your keyboard.
+1. In the playlist page, click **Edit playlist**. A prompts opens asking you to confirm the action.
```
[the remark this answers is not in the document]
Comment (achatterjee-grafana) [reply docs/sources/dashboards/playlist.md:85]: Keeping this as is.

[the remark this answers is not in the document]
Comment (GrafanaWriter) [reply docs/sources/dashboards/playlist.md:85]: Totally fine. That's why it's a suggestion :)

**Pull request merged**